### PR TITLE
use client package in cfsctl

### PR DESCRIPTION
**Pull request by yutongp**
- use client package in cfsctl
- add `Stats` func in client

### CODE REVIEW

File: client/client.go [hunk 1]
```diff
@@ -88,10 +91,25 @@ func (c *Client) Mkdir(ctx context.Context, name string, all bool) error {
...
 }
 
+func (c *Client) Stats(ctx context.Context) (string, error) {
+	reply, err := c.statsClient.ContainerInfo(ctx, &pb.ContainerInfoRequest{})
+
+	if err != nil {
+		return "", err
+	}
+	if reply.Error != "" {
+		return reply.Info, errors.New(reply.Error)
```
Comment (yutongp) [inline client/client.go:101]: Is there a chance we can have only one basic Error type in pb for both file.pb and stats.pb and further pb?

File: cfsctl/cfs_command.go [hunk 2]
```diff
@@ -38,11 +38,11 @@ func addCommand() {
...
 }
 
-func setUpGrpcClient() *grpc.ClientConn {
+func setUpClient() *client.Client {
 	// Set up a connection to the server.
-	conn, err := grpc.Dial(address)
+	c, err := client.New(address)
 	if err != nil {
-		log.Fatalf("Cannot connect: %v", err)
+		log.Fatalf("Cannot create CFS client: %v", err)
```
Comment (xiang90) [inline cfsctl/cfs_command.go]: Cannot create cfs client:

File: client/client.go (hunk 1 above)
Comment (xiang90) [inline client/client.go:101]: probably... I am still not sure about that.

**Comment by xiang90** [comment]
LGTM

**Pull request merged**